os/include/tinyara/serial/tioctl.h

```diff
@@ -202,6 +202,7 @@
 #define TIOCCLRSIG      _TIOC(0x002f)	/* Clear signal to handle Interrupt */
 #define TIOCTXSTATE     _TIOC(0x0030)	/* Get State of TX */
 #define TIOCRXSTATE     _TIOC(0x0031)	/* Get State of RX */
+#define TIOCLOOPBACK    _TIOC(0x0032)	/* Enable/Disable loopback mode */
```

Comment:
For most of constant value, we usually follow linux's, there's TIOCM_LOOP in linux. Please check it.